### PR TITLE
Add definition tooltip

### PR DESCRIPTION
**Pull request by kingtraceyj**
#### Changelog

**New**

• Added definition tooltip docs into tooltip from toggletip

**Changed**

• Updated images
• Updated style tables and images

### CODE REVIEW

**Review by dakahn — APPROVED** [review]

**Review by laurenmrice — APPROVED** [review]
✅

File: src/pages/components/tooltip/style.mdx [hunk 1]
```diff
@@ -52,6 +64,36 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
   button tooltip
 </Caption>
 
+### Definition toggletip
+
+| element   | property         | color token           |
```
Comment (laurenmrice) [inline src/pages/components/tooltip/style.mdx:69]: The column header names should be capitalized.

File: src/pages/components/tooltip/style.mdx [hunk 2]
```diff
@@ -52,6 +64,36 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
   button tooltip
 </Caption>
 
+### Definition toggletip
```
Comment (laurenmrice) [inline src/pages/components/tooltip/style.mdx:67]: This is technically a tooltip now right? Not toggletip?

File: src/pages/components/tooltip/style.mdx [hunk 3]
```diff
@@ -11,13 +11,20 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
...
-| Container             | background-color | `$background-inverse` |
-| Text                  | color            | `$text-inverse`       |
+| Element        | Property         | Color token           |
+| -------------- | ---------------- | --------------------- |
+| Label          | text color       | `$text-secondary`     |
+| Trigger button | svg              | `$icon-secondary`     |
+| Container      | background-color | `$background-inverse` |
+| Text           | color            | `$text-inverse`       |
+
+#### Interactive states
```
Comment (laurenmrice) [inline src/pages/components/tooltip/style.mdx:21]: The "Interactive states" heading seems to have the incorrect padding between the heading and the table below it.

**Review by laurenmrice — CHANGES_REQUESTED** [review]
The image for Definition tooltip should be updated to cahnge the label in the images from toggletip to `tooltip`.
Screen Shot 2022-01-18 at 2 37 42 PM

**Comment by kingtraceyj** [comment]
@laurenmrice i added breaks after the interactive state headings ... let me know if that's what it's supposed to look like! it's better than before but now it seems like such a large break.

**Review by laurenmrice — APPROVED** [review]

**Pull request merged**